docs/readme-template.md

```diff
@@ -125,8 +128,8 @@ For this example, we'll create a new "Banner" component.
 
 1. Next, create the Banner component:
 
-   ```jsx
-   // src/components/banner.js
+   ```jsx fileExt
```

Comment:
Haven't checked yet, but I'm guessing the `jsx` syntax highlighting on GitHub works for tsx syntax, but if it doesn't 100%, I would guess it'd work the other way around. Then again, maybe it doesn't matter because there are significant changes to the code examples' contents

Reply:
Yeah I thought about that, but didn't think it was worth the trouble because the syntax highlighting won't be different. If we did add in an interface for the props, that wouldn't be highlighted the same way with `jsx` and `tsx` but I figure people will be able to figure out adding:

```
interface BannerProps {
    heading: string;
    text: string;
}
```

without us holding their hands on this one.